api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -71,6 +71,10 @@
               .setName(AccessModuleName.RT_COMPLIANCE_TRAINING)
               .setExpirable(true)
               .setBypassable(true),
+          new DbAccessModule()
+              .setName(AccessModuleName.CT_COMPLIANCE_TRAINING)
+              .setExpirable(true)
+              .setBypassable(true),
```

Comment:
false

Reply:
Oops I commented on the wrong line :)

https://github.com/all-of-us/workbench/pull/5516